### PR TITLE
added submodules from pcl_conversions and perception_pcl

### PR DESCRIPTION
**Pull request by mfehr**
@eggerk

### CODE REVIEW

**Comment by mfehr** [comment]
@ffurrer does this PR still make sense?

**Comment by ffurrer** [comment]
yes in general I think this makes a lot of sense, as this guarantees that people are also using the packages depending on this version of pcl.

**Comment by mfehr** [comment]
@ffurrer testing it now

**Comment by mfehr** [comment]
this would need some refactoring on the catkin dependencies part, let's wait until we actually need them

File: pcl_catkin/CMakeLists.txt [hunk 1]
```diff
@@ -19,6 +19,7 @@ string(TOUPPER ${CMAKE_BUILD_TYPE} UC_BUILD_TYPE)
 # as this can introduce different vectorization settings in different parts
 # of the code. So we manually set the compiler flags here to avoid this issue.
 include("${CMAKE_SOURCE_DIR}/cmake/pcl_find_sse.cmake")
+message("${CMAKE_SOURCE_DIR}/cmake/pcl_find_sse.cmake")
```
Comment (ntonci) [inline pcl_catkin/CMakeLists.txt]: rm

**Pull request merged**